tests/h/views/activity_test.py

```diff
@@ -266,6 +261,35 @@ def pyramid_request(self, group, pyramid_request):
     def routes(self, pyramid_config):
         pyramid_config.add_route('activity.search', '/search')
 
+class TestUserSearch(object):
+    def test_it_returns_404_when_feature_turned_off(self,
+                                                    pyramid_request):
+        pyramid_request.feature.flags['search_page'] = False
+
+        with pytest.raises(httpexceptions.HTTPNotFound):
+            activity.user_search(pyramid_request)
+
```

Comment:
Should only have one new line between methods, as opposed to the two blank lines between top level items in a file